src/CMakeLists.txt

```diff
@@ -53,6 +54,12 @@ if(NVIDIA_SUPPORT)
   target_sources(nvtop PRIVATE extract_gpuinfo_nvidia.c)
 endif()
 
+if(ASCEND_SUPPORT)
+  target_sources(nvtop PRIVATE extract_gpuinfo_ascend.c)
+  set(DCMI_LIBRARY_PATH /usr/local/Ascend/driver/lib64/driver)
```

Comment:
Is the path guaranteed to be installed there?
Otherwise I could write a simple FindAscend.cmake to look for `Ascend/driver/lib64/driver/libcmi.so`  using [find_library](https://cmake.org/cmake/help/latest/command/find_library.html) or [find_file](https://cmake.org/cmake/help/latest/command/find_file.html#find-file)

Reply:
In most cases it is installed in this path by default (unless in some very unusual cases may need to change manually), but I think it will be better if there is a script to help to find it :)